packages/@extension-kid/notification-service/NotificationService.ts

```diff
@@ -0,0 +1,48 @@
+import { injectable, inject, named } from "inversify";
```

Comment:
Would love to see a README in the package with some design decisions that went into the package and some documentation around how to design an extension for the service.

Reply:
I started a README, I'll expand it more today.